### ISSUE TITLE
Skaffold init in force mode may still run interactively

### ISSUE THREAD

**Issue by ivanporty** [GoogleContainerTools/skaffold#2235]
Cloud Code IJ uses `skaffold init --force` to silently generate configuration in default mode. Here is what we have for one of the project configurations (this is actually not a real K8s project, rather a random bunch of pseudo Dockerfiles and K8s resources for testing, but detection in principle is correct), but --force mode still runs interactively for some reason:

```
skaffold init --force
? Choose the dockerfile to build image mhausenblas/simpleservice:0.5.0  [Use arrows to move, type to filter]
❯ app-engine/java/build/resources/main/fileTemplates/internal/JarDockerfile.gaedocker.ft
  app-engine/java/build/resources/main/fileTemplates/internal/WarDockerfile.gaedocker.ft
  app-engine/java/resources/fileTemplates/internal/JarDockerfile.gaedocker.ft
  app-engine/java/resources/fileTemplates/internal/WarDockerfile.gaedocker.ft
  None (image not built from these sources)

```

skaffold version: v0.29.0

**Comment by nkubala** [comment]
@ivanporty the documentation for the `--force` flag is a little misleading here. if skaffold finds multiple images and Dockerfiles when running `init`, it can't know which Dockerfile is meant to build which image, so it has no choice but to prompt the user for this before it can generate a skaffold.yaml. so in the case you provided, `skaffold init --force` shouldn't really work.

I guess the question is, what UX do you think users would expect here? to me running with the `--force` flag should just fail out when it can't succeed without user input. maybe there's an easy flow for you guys to collect this input on the IDE side? the image/dockerfile pairs can be passed on the CLI (e.g. `skaffold init --force --artifact=/web/Dockerfile.web=gcr.io/web-project/image`), so if you had that information ahead of time this could still be useful for you. WDYT?

**Comment by TadCordle** [comment]
Fixed with #3252
